tests/end-to-end/upgrade/chart/upgrade/templates/pod.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-{{ template "fullname" . }}-prepare-data
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
```

Comment:
label `"kyma-project.io/upgrade-e2e-test": "prepareData"` is missing but it's required because of the tests.